content/applications/productivity/to_do.rst

```diff
@@ -0,0 +1,148 @@
+
+=====
+To-do
+=====
+
+The To-Do app assists users in efficiently organizing and managing their work. It enables seamless
+collaboration and streamlined workflows across different aspects of the business, offering features
+such as creating and assigning tasks, setting deadlines, creating memos and to-do lists, monitoring
+progress, and much more.
+
+Configuration
+=============
+
+If To-Do is not yet installed on your database, go to the :guilabel:`Apps`, search for it, and click
```

Comment:
If :guilabel:'To-Do' is not yet installed on your database ...

Reply:
no it's not necessary as we don't show something on the screen